optimize/backend/pom.xml

```diff
@@ -166,16 +190,20 @@
         </exclusion>
       </exclusions>
     </dependency>
+    -->
+    <!-- unused/declared
     <dependency>
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-jetty</artifactId>
       <version>${spring.boot.version}</version>
     </dependency>
+    -->
```

Comment:
Removing all of these Spring ones are interesting. We should heavily test this to make sure we aren't breaking things